tests/integrations/test_lightning.py

```diff
@@ -188,7 +188,7 @@ def __init__(self):
             self.metric_epoch = SumMetric()
             self.sum = torch.tensor(0.0)
 
-        def on_epoch_start(self):
+        def on_train_epoch_start(self):
```

Comment:
From which version this is so so set min PL version right...

Reply:
```python
RuntimeError: The `LightningModule.on_epoch_start` hook was removed in v1.8. Please use `LightningModule.on_<train/validation/test>_epoch_start` instead.
```

Reply:
but we could always do it like this, no matter the actual version.

Reply:
That is fine I was wondering if this edit would be compatible also with 
https://github.com/Lightning-AI/metrics/blob/8fc6de84d804dc7db9f21a307d2adef3029e6591/requirements/integrate.txt#L1

Reply:
seem the oldest was missed, so fixing it here: #1310